ddtrace/contrib/pytest/plugin.py

```diff
@@ -207,8 +207,8 @@ def pytest_runtest_makereport(item, call):
     else:
         if xfail and not has_skip_keyword:
             # XPass (strict=True) are recorded failed by pytest, longrepr contains reason
-            span.set_tag(test.XFAIL_REASON, result.longrepr)
```

Comment:
I don't think `.longrepr` is the correct attribute to access: it's not always set and it doesn't seem to carry the reason